tensorflow/core/ops/parsing_ops.cc

```diff
@@ -48,6 +48,20 @@ output: A Tensor with one more dimension than the input `bytes`.  The
   of `bytes` divided by the number of bytes to represent `out_type`.
 )doc");
 
+REGISTER_OP("DecodeCompressed")
+    .Input("bytes: string")
+    .Output("output: string")
+    .Attr("compression_type: string = ''")
+    .SetShapeFn(shape_inference::UnchangedShape)
+    .Doc(R"doc(
+Reinterpret the bytes of a string as a vector of numbers.
```

Comment:
Didn't quite follow the "vector of numbers" comment here.

Reply:
@asimshankar Sorry the doctoring was not corrected. The PR has been updated. Please take a look.